astroid/nodes/node_classes.py

```diff
@@ -61,6 +62,9 @@
 else:
     from typing_extensions import Literal
 
+if TYPE_CHECKING:
+    from astroid.scoped_nodes import LocalsDictNodeNG
```

Comment:
```suggestion
    from astroid.nodes import LocalsDictNodeNG
```
Or if really necessary:
```suggestion
    from astroid.nodes.scoped_nodes import LocalsDictNodeNG
```
Because ``astroid.scoped_nodes`` is deprecated and will be removed in 3.0

Reply:
`LocalsDictNodeNG` was not imported in `astroid.nodes`. I added it there and added an import from `astroid.nodes`.

Reply:
Thank you for fixing the underlying problem, it will prevent problems during 3.0 migrations !